esm/interpreter/micropython.js

```diff
@@ -16,6 +17,16 @@ export default {
         if (config.files) await fetchFiles(this, interpreter, config.files);
         if (config.fetch) await fetchPaths(this, interpreter, config.fetch);
         if (config.js_modules) await fetchJSModules(config.js_modules);
+        //Install Micropython Package
+        const enc = new TextEncoder()
+        this.writeFile(interpreter, './mip.py', enc.encode(mipSrc))
```

Comment:
I think this is needed **only** if there is a `config.packages` unless you are suggesting `import mip` should be enabled by default ... is that the case?